### ISSUE TITLE
[React.Suspense]: Cannot convert a Symbol value to a string

### ISSUE THREAD

**Issue by quangrau** [adriantoine/enzyme-to-json#116]
Hello,

I'm getting this error when using new React's feature `Suspense/lazy`.

Here's is my `AppContent.jsx` file:
```jsx
      <React.Suspense fallback={<Spinner />}>
        <Switch>
          {ROUTES.map(route => (
            <Route
              key={route.pathname}
              path={route.pathname}
              exact={route.exact}
              component={route.component}
            />
          ))}
        </Switch>
      </React.Suspense>
```

And here's `AppContent.test.jsx` file:
```jsx
describe('AppContent', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<AppContent />);
    expect(wrapper).toMatchSnapshot();
  });
});
```

My `package.json`:
```json
    ...
    "react": "^16.6.0",
    "react-dom": "^16.6.0",
},
"devDependencies": {
    "jest: "23.6.0",
    "enzyme": "3.7.0",
    "enzyme-adapter-react-16": "1.6.0",
    "enzyme-to-json": "3.3.4"
}
```

Does anyone get this error like this? I thought this is the same issue which is happened with `React.Fragment` before. Looking forward to hearing more advice from you guys.

Cheers.

**Comment by ersel** [comment]
We started seeing this on latest version as well.

**Comment by EugeneBugor** [comment]
This issue was fixed in the latest version of the package.

**Comment by VincentLanglet** [comment]
> This issue was fixed in the latest version (3.3.5) of the package.

So I can close the issue
